frontend/src/context/AuthContextProvider.tsx

```diff
@@ -24,14 +24,14 @@ export const AuthContextProvider: React.FC = ({ children }) => {
   const logout = useCallback(async () => {
     localStorage.clear();
     await Auth.signOut();
+    window.location.reload();
   }, []);
 
   const handleError = useCallback(
     async (e: Error) => {
       if (e.message.includes('401')) {
         // Unauthorized, log out user
         await logout();
-        window.location.reload();
```

Comment:
Should we keep this line in? (this way, when the user's JWT expires, they'll have the page refresh)